src/test/regress/expected/metadata_sync_helpers.out

```diff
@@ -987,6 +987,7 @@ BEGIN TRANSACTION ISOLATION LEVEL READ COMMITTED;
 ERROR:  Node with group id 123123123 for shard placement xxxxx does not exist
 ROLLBACK;
 -- create a volatile function that returns the local node id
+SET citus.enable_metadata_sync TO OFF;
```

Comment:
why do we need to disable metadata sync for this one? I couldn't follow

Reply:
Since we are on the worker node.